app/views/paymentMethods/creditCard.html

```diff
@@ -82,7 +82,97 @@ <h4 translate>Billing Address</h4>
         />
       </div>
     </div>
-
+    <div class="form-group">
+      <div class="col-sm-5 stacked-spacing-col-sm">
+        <label translate>City</label>
+        <input
+          type="text"
+          ng-model="currentPayment.creditCard.billingCity"
+          class="form-control"
+        />
+      </div>
+      <div
+        class="col-sm-4 stacked-spacing-col-sm"
+        ng-show="creditCardCountry !== 'US'"
+      >
+        <label translate>State/Region</label>
+        <input
+          type="text"
+          ng-model="currentPayment.creditCard.billingState"
+          class="form-control"
+        />
+      </div>
+      <div
+        class="col-sm-4 stacked-spacing-col-sm"
+        ng-show="creditCardCountry === 'US'"
+      >
+        <label translate>State</label>
+        <select
+          ng-model="currentPayment.creditCard.billingState"
+          class="form-control"
+        >
+          <option value="AL">Alabama</option>
```

Comment:
Can these be pulled from the library?

Reply:
Yeah, I forgot about this section. Updating now!